README.md

```diff
@@ -94,11 +92,26 @@ pipenv install
       - `pipenv install windows-curses`
 </details>
 
+<details>
+  <summary>If you get `python : The term 'python' is not recognized` even after installing Python 3.8 </summary>
+ 
+  1. Open the Start menu and enter `PATH`, then press Enter
+  2. Click the `Environment Vairables...` button
+  3. In the `User variables for user` section, click the variable `Path`, then click the `Edit...` button
+  4. A new window called `Edit environment variable` will pop up, ensure `C:\Users\user\AppData\Local\Microsoft\WindowsApps` exists in this list, where `user` is your username in Windows
+  5. If that path is not in the list, add it using the `New` button
+  6. Remove all other PATHs which contain anything to do with Python, Pip, or Idle by clicking them, then clicking the `Delete` button
+  7. Click `OK` to close the `Edit environment variable` window
+  8. Click `OK` to close the `Environment Variables` window
+  9. Click `OK` to close the `System Properties` window
+
+ </details>
+
 ### CAPTCHA
 
-To enable CAPTCHA support, follow these steps.
+This bot is equipped with the ability to resolve simple reCAPTCHA on its own without the use of a third party reCAPTCHA solver.  As long as you follow the recommendation in the usage section below, it is extremely likely that the bot will succesfully complete the full auto-checkout process.  With that said, there is a very small possibility that you will be presented with a more advanced reCAPTCHA requiring solving image tasks that the bot cannot handle natively.  If you would like to have a backup in place for that possibility, you have that option by following the steps below.   
 
-1. Download the [ReCaptcha Solver](https://addons.mozilla.org/en-US/firefox/addon/recaptcha-solver) extension using the link https://addons.mozilla.org/firefox/downloads/file/3423472/recaptcha_solver-5.7-fx.xpi. If you open this link in Firefox, it will automatically attempt to add the extension to your browser. That's not what we want. Right click the link and select "Save link as" to download the *.xpi file or try opening the link in a browser other than Firefox.
+1. Download the [ReCaptcha Solver](https://addons.mozilla.org/en-US/firefox/addon/recaptcha-solver) extension using the link https://addons.mozilla.org/firefox/downloads/file/3423472/recaptcha_solver-5.7-fx.xpi. If you open this link in Firefox, it will automatically attempt to add the extension to your browser. That's not what we want. Right click the link and select "Save link as" to download the *.xpi file. Try using the link in a browser other than Firefox.
```

Comment:
```suggestion
1. Download the [ReCaptcha Solver](https://addons.mozilla.org/en-US/firefox/addon/recaptcha-solver) extension using the link https://addons.mozilla.org/firefox/downloads/file/3423472/recaptcha_solver-5.7-fx.xpi. If you open this link in Firefox, it will automatically attempt to add the extension to your browser. That's not what we want. Right click the link and select "Save link as" to download the *.xpi file or try opening. Try using the link in a browser other than Firefox.
```